indico/modules/designer/operations.py

```diff
@@ -0,0 +1,57 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2017 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from flask import session
+
+from indico.modules.designer.models.templates import DesignerTemplate
+from indico.modules.events import EventLogKind, EventLogRealm
+
+
+def update_template(template, title, data, backside_template_id=None, disassociate_backsides=False,
+                    clear_background=False):
+    """Update an existing template.
+
+    :param template: The template to be updated
+    :param title: An `EventType` value
+    :param data: A dict containing the template data (width, height, items, etc)
+    :param backside_template_id: The ID of the template used as a backside
+    :param disassociate_backsides: Whether to remove the specific template from
+                                   being used as the backside of others
+    :param clear_background: Whether to remove the background image of the
+                             template
+    """
+    template.title = title
+    template.backside_template = DesignerTemplate.get(backside_template_id) if backside_template_id else None
+    template.data = dict({'background_position': 'stretch', 'items': []}, **data)
+
+    if disassociate_backsides:
```

Comment:
I don't like the idea of making this an argument, even less letting the client specify its value. I'd rather check where whether the backside dimensions don't match and in that case disassociate it.